#/usr/bin/env python

PACKAGE = "laser_daten"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("max_pose_distance", double_t, 0, "maximale x abweichung", 0.1, 0.000, 1.000)
gen.add("max_pose_distance_s_e", double_t, 0, "maximale z_quaternion abweichung", 0.1, 0.000, 1.0000)
gen.add("max_pose", double_t, 0, "maximale x abweichung", 0.1, 0.000, 1.000)


exit(gen.generate(PACKAGE, "laser_daten", "SLAMParam"))
